app/views/adoption_requests/new.html.erb

```diff
@@ -1,4 +1,5 @@
-<% provide(:title, "Adoption Request Form") %>
+<% page_title! 'Adoption Request Form' %>
```

Comment:
What's the benefit of using the `page_title_helper` gem instead of the `provide` helper?